src/main/java/org/terasology/potions/PotionStatusUISystem.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.potions;
+
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.EventPriority;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterMode;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.input.ButtonState;
+import org.terasology.input.binds.inventory.InventoryButton;
+import org.terasology.logic.delay.DelayManager;
+import org.terasology.logic.delay.PeriodicActionTriggeredEvent;
+import org.terasology.potions.component.PotionEffect;
+import org.terasology.potions.events.DrinkPotionEvent;
+import org.terasology.registry.In;
+import org.terasology.rendering.nui.NUIManager;
+
+import java.util.ArrayList;
+
+
+@RegisterSystem(RegisterMode.CLIENT)
+public class PotionStatusUISystem extends BaseComponentSystem {
+
+    @In
+    private NUIManager nuiManager;
+    @In
+    private DelayManager delayManager;
+    @In
+    private EntityManager entityManager;
+
+    private PotionStatusScreen screen;
+
+    private ArrayList<Long> durations = new ArrayList<>();
+    private ArrayList<String> effects = new ArrayList<>();
+
+    private boolean isScreenVisible = false;
+
+    private int updateRate = 200;
+
+    @Override
+    public void initialise() {
+    }
+
+    @ReceiveEvent(priority = 110)
+    public void inventoryToggled(InventoryButton event, EntityRef entity) {
+        if (event.getState() == ButtonState.DOWN) {
+            nuiManager.toggleScreen("potions:potionStatusScreen");
+            isScreenVisible = nuiManager.isOpen("potions:potionStatusScreen");
+            updateScreen();
+        }
+    }
+
```

Comment:
Add a method here so that pressing the PauseButton will close out the active potion effects window. This is so that it mimics the behavior of the inventory screen.

```
    @ReceiveEvent(priority = 110)
    public void inventoryToggled(PauseButton event, EntityRef entity) {
        if (nuiManager.isOpen("potions:potionStatusScreen")) {
            nuiManager.closeScreen("potions:potionStatusScreen");
            isScreenVisible = nuiManager.isOpen("potions:potionStatusScreen");
            updateScreen();
        }
    }
```